layouts/shortcodes/alert.html

```diff
@@ -0,0 +1,49 @@
+{{ $color := index . 0 }}
```

Comment:
My main reasoning was that we might want to call a banner on a specific page inside the page's markdown file.

I have looked into it, but it seems that you can't call a partial from markdown, and you can't call a shortcode from a layout.

Rather than duplicating the code though, I'll instead have the shortcode call the partial. Keeping both accessible, but removing that duplication (which I hated anyway)